### ISSUE TITLE
Text is not showing properly

### ISSUE THREAD

**Issue by hassadee** [VeejayAdolf-Lemita/sdi_front-end-challenge-1#2]
The text content is not showing properly.

"TMSPH launches Toyota Rent&#x3B1;Car"

It's supposed to be rendered as

"TMSPH launches Toyota RentαCar"

Could you fix that?

**Comment by VeejayAdolf-Lemita** [comment]
Yes sir we can't change the data here is my final fix I added a decoder function which converts the json HTML entities. Please check my latest push sir. Thank you
